components/runners/ambient-runner/ambient_runner/endpoints/auto_analysis.py

```
@@ -0,0 +1,299 @@
+
+    for filename in always_read:
+        filepath = repo_path / filename
+        if not filepath.exists() or not filepath.is_file():
+            continue
+        try:
+            content = filepath.read_text(errors="replace")
+            if len(content) > 3000:
+                content = content[:3000] + "\n... (truncated)"
+            parts.append(f"## {filename}\n```\n{content}\n```\n")
+        except Exception:
+            continue
```

Ruff (0.15.10)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)